ProcessesApi/V1/Domain/Finance/PaymentAgreement.cs

```diff
@@ -0,0 +1,8 @@
+namespace ProcessesApi.V1.Domain.Finance
```

Comment:
Are there any shared package that you could potentially use?

Reply:
No, this API uses Ruby.